.gitignore

```diff
@@ -19,7 +19,7 @@ bin
 .project
 .classpath
 
-
+.history
```

Comment:
Can you revert this one line change please?

Reply:
Done